include/RAJA/MemUtils_CPU.hxx

```diff
@@ -58,10 +58,31 @@
 
 #include "RAJA/int_datatypes.hxx"
 
+#include <stddef.h>
```

Comment:
Although we don't have a rule about this, should we be using cstddef or stddef.h?